scripts/jenkins/ci1/freshvm

```diff
@@ -20,16 +20,13 @@ if ! [ -f $p/$flavor.img -o -f $p/$flavor.qcow2 ]; then
 fi
 
 virsh destroy $n
-if [ "$n" = "cleanvm" ] ; then
-    dd if=/dev/zero of=/dev/vg0/$n-data count=16
-    if [ -e $p/$flavor.qcow2 ] ; then
-        time qemu-img convert $p/$flavor.qcow2 /dev/vg0/$n
-    else
-        time dd if=$p/$flavor.img of=/dev/vg0/$n bs=64k
-    fi
+dd if=/dev/zero of=/dev/vg0/$n-data count=16
```

Comment:
I found `fallocate` a lot faster than `dd`. In this case maybe is not important, but if there are other `dd` we can think to replace them